.github/workflows/publish.yml

```
@@ -0,0 +1,17 @@
+
+jobs:
+  publish-to-gallery:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Publish
+        env:
+          NUGET_KEY: ${{ secrets.NUGET_KEY }}
+        shell: pwsh
+        run: |
+          Publish-Module -Path ./ps-copyid/ -NuGetApiKey $env:NUGET_KEY -Repository PSGallery
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)